src/options/options.html

```diff
@@ -103,7 +103,15 @@ <h1 data-i18n="__MSG_titleUnicodeAutocorrection__">Unicode autocorrection</h1>
 							<li>
 								<div class="line">
 									<input class="setting save-on-change" type="checkbox" id="autocorrectUnicodeFracts" data-optiongroup="autocorrect" name="autocorrectUnicodeFracts">
-									<label data-i18n="__MSG_optionConvertFractions__" data-opt-i18n-keep-children for="autocorrectUnicodeFracts">Convert fractions and mathematical constants to Unicode characters</label>
+									<label data-i18n="" data-opt-i18n-keep-children for="autocorrectUnicodeFracts">Convert fractions to Unicode characters</label>
```

Comment:
Well could you please add the i18n entries to the English `messages.json` and add them here?

This makes it easier to localize later.